src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -49,6 +49,16 @@
   product(bool, StressGCM, false, DIAGNOSTIC,                               \
           "Randomize instruction scheduling in GCM")                        \
                                                                             \
+  product(bool, StressIGVN, false, DIAGNOSTIC,                              \
+          "Randomize worklist traversal in IGVN")                           \
+                                                                            \
+  product(bool, GenerateStressSeed, false, DIAGNOSTIC,                      \
```

Comment:
Is this flag really required? We could simply generate the seed if StressSeed has not been specified on the command line (see `FLAG_IS_DEFAULT` macro).

Reply:
Agree.

Reply:
Done. I didn't know about `FLAG_IS_DEFAULT`, thanks!